srcpkgs/bloaty/template

```diff
@@ -12,12 +12,11 @@ homepage="https://github.com/google/bloaty"
 distfiles="https://github.com/google/bloaty/releases/download/v${version}/bloaty-${version}.tar.bz2"
 checksum=e1cf9830ba6c455218fdb50e7a8554ff256da749878acfaf77c032140d7ddde0
 
-case "$XBPS_TARGET_MACHINE" in
-	armv6*) broken="undefined reference to __atomic_fetch_add_8'" ;;
-esac
-
 if [ "$CROSS_BUILD" ]; then
 	hostmakedepends+=" protobuf"
+	makedepends+=" libatomic-devel"
+	# Force linking of libatomic ; armv6l will fail without
+	export CMAKE_CXX_FLAGS="-L${XBPS_CROSS_BASE}/usr/lib -lm -latomic"
```

Comment:
Wouldn't that break native compile?

Reply:
I wouldn't expect it to do so, since it is for cross-compiling and you can use LDFLAGS in C/CXXFLAGS as well.